### DISCUSSION TITLE
FireAlpaca Icon?

### DISCUSSION THREAD

**Discussion by Murplyy** [elrumo/macOS_Big_Sur_icons_replacements#1381] [Icon request]
Would love an icon for FireAlpaca! (A drawing software)
For reference >> https://imgur.com/a/xt7P5CZ

Without a white background would be preferable!

**Comment by 9hyz** [comment]
Hi, not sure if this was the sort of thing your were looking for but a bit of mucking around with Vectornator led me to this:

> ![firealpaca](https://user-images.githubusercontent.com/88746918/181770683-ce60eabf-65a8-4e5b-b8a0-f1649b140d15.png)

[Amateurish but fun to make!](https://github.com/dative9/firealpaca---macOsX/releases/tag/iconset)
